bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectEVCharger.xml

```diff
@@ -0,0 +1,135 @@
+			</channel>
+			<channel id="charger-voltage-phase1" typeId="system.electric-voltage">
+				<label>Charger Voltage Phase 1</label>
+				<description>Voltage of the charger's phase 1</description>
+			</channel>
+			<channel id="charger-voltage-phase2" typeId="system.electric-voltage">
+				<label>Charger Voltage Phase 2</label>
+				<description>Voltage of the charger's phase 2</description>
+			</channel>
+			<channel id="charger-voltage-phase3" typeId="system.electric-voltage">
+				<label>Charger Voltage Phase 3</label>
+				<description>Voltage of the charger's phase 3</description>
+			</channel>
+
+			<channel id="charger-external-current-phase1" typeId="system.electric-current">
+				<label>Charger External Current Phase 1</label>
+				<description>Current from the provider phase 1</description>
+			</channel>
+			<channel id="charger-external-current-phase2" typeId="system.electric-current">
+				<label>Charger External Current Phase 2</label>
+				<description>Current from the provider phase 2</description>
+			</channel>
+			<channel id="charger-external-current-phase3" typeId="system.electric-current">
+				<label>Charger External Current Phase 3</label>
+				<description>Current from the provider phase 3</description>
+			</channel>
+			<channel id="charger-external-power-phase1" typeId="system.electric-power">
+				<label>Charger External Power Phase 1</label>
+				<description>Power from the provider phase 1</description>
+			</channel>
+			<channel id="charger-external-power-phase2" typeId="system.electric-power">
+				<label>Charger External Power Phase 2</label>
+				<description>Power from the provider phase 2</description>
+			</channel>
+			<channel id="charger-external-power-phase3" typeId="system.electric-power">
+				<label>Charger External Power Phase 3</label>
+				<description>Power from the provider phase 3</description>
+			</channel>
+			<channel id="charger-external-total-power" typeId="system.electric-power">
+				<label>Charger External Total Power</label>
+				<description>Total power from the provider</description>
+			</channel>
+
+			<channel id="charger-plug-temperature" typeId="charger-temperature">
+				<label>Charger Plug Temperature</label>
+				<description>Temperature of the charger's plug</description>
+			</channel>
+			<channel id="charger-internal-temperature" typeId="charger-temperature">
+				<label>Charger Internal Temperature</label>
+				<description>Internal temperature on the board of the charger</description>
+			</channel>
+			<channel id="charger-mode" typeId="charger-mode">
+				<label>Charger Workmode</label>
+				<description>Charger Workmode</description>
+			</channel>
+			<channel id="charger-state" typeId="charger-state">
+				<label>Charger State</label>
+				<description>Charger State</description>
+			</channel>
+
+			<channel id="last-update-time" typeId="last-retrieve-time-stamp"/>
+
+			<channel id="raw-data" typeId="raw-data-type"/>
```

Comment:
See other comment about raw channel

Reply:
I'll leave this open while we discuss the usage of this channel in the other comment